invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -49,6 +59,8 @@ const generateFieldProps = (
     required: fieldSchema.required,
     disabled: fieldSchema.readOnly || (fieldSchema.createOnly && !isCreate),
```

Comment:
if we're checking for `fieldSchema` in other calls (e.g. `fieldSchema?.metadata?.options`) then we should also check it here.

On the other hand, if we're assuming `fieldSchema exists` then the optional chain operator `?` only needs to be applied to nested attributes (e.g. `fieldSchema.metadata?.options`

Reply:
Thank you for the notice!
Result of digging a bit deeper in the code: schema itself is required, otherwise no fields rendered. 
If some of the schema fields are not specified, the default `marshmallow` schema values are used corresponding to field type.
So I would leave the `?` operator only on the nested fields, WDYT?
Also, maybe there was some reason to do it that I don't see. @kpsherva can you help?

Reply:
If  `fieldSchema` is defined then yes, I agree and you can use the chain operator only for nested attributes. On the other hand if `fieldSchema` is not defined, you must use it **always**. 

For example:

```javascript
// fieldSchema is not instantiated
let fieldSchema = undefined;  

console.log(fieldSchema.metadata); // throws an exception
console.log(fieldSchema?.metadata); // logs undefined
console.log(fieldSchema?.metadata?.value); // logs undefined

// fieldSchema is instantiated
fieldSchema = {}; 
console.log(fieldSchema.metadata); // logs undefined
console.log(fieldSchema?.metadata); // logs undefined
console.log(fieldSchema.metadata?.value); // logs undefined
console.log(fieldSchema.metadata?.value.deeply_nested); // logs undefined
```

You can try the snippet in https://jsfiddle.net/

Reply:
If schema is not defined, we are getting an error on the backend side (`'NoneType' object is not callable`) [here](https://github.com/inveniosoftware/invenio-administration/blob/a63feced37037bd586be84ec34e5f2a054440f1b/invenio_administration/views/base.py#L188)